### PR TITLE
docs(three-d-node-tree): rewrite stories in new format

### PR DESCRIPTION
**Pull request by amevck**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-549.gearboxjs.preview.cogniteapp.com

File: src/components/ThreeDNodeTree/stories/basic.story.mdx [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+<Meta title="Others|ThreeDNodeTree" decorators={decorators} />
+
+# ThreeDNode Tree
+
+Visualize parent-child relationship of 3D nodes in a tree structure. It will initially fetch all available root from a model. When a node is expanded, its children will be fetched dynamically. It behaves the same with Asset Tree.
+
+To retrieve a tree you need to provide:
+
+- `modelId` – you can find it using `sdk.ThreeD.listModels()` call
+- `revisionId` – you can find it via `sdk.ThreeD.listRevisions(modelID)` call
```
Comment (polomani) [inline src/components/ThreeDNodeTree/stories/basic.story.mdx]: this is outdated, please update it according to:
https://github.com/cognitedata/cognite-sdk-js/blob/v1/guides/MIGRATION_GUIDE_1xx_2xx.md

ask if not clear

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-549.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-549.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-549.gearboxjs.preview.cogniteapp.com

**Pull request merged**